tests/PHPStan/Analyser/LegacyNodeScopeResolverTest.php

```diff
@@ -5271,30 +5271,6 @@ public function dataFunctions(): array
 				'bool',
 				'$versionCompare8',
 			],
-			[
-				'int<0, max>',
-				'$mbStrlenWithoutEncoding',
```

Comment:
Please do not remove existing tests.

Reply:
I didn't remove them, I rewrote them using NodeScopeResolverTest instead of the legacy one.
Here's this test: https://github.com/phpstan/phpstan-src/pull/1211/files#diff-a391b3fee24595d24e76c2bf53174492bd40ef80c690ead5fdbf8736df513f48R23